index.html

```diff
@@ -48,198 +61,254 @@ <h2 class="describe">I am a Software Developer</h2>
-      </li>
-    </ul>
-    <ul class="list">
-      <li class="list">
-        <a class="skillset" href="#">Javascript</a>
-        <a class="skillset" href="#">Ruby</a>
-        <a class="skillset" href="#">Html</a>
-        <a class="skillset" href="#">Css</a>
-      </li>
-    </ul>
+    <div class="col4">
+      <ul class="language">
+        <a class="icon" href="#"><img src="images/language.png" alt="" /></a>
+        <li>
+          <a class="tech">Languages</a>
+        </li>
+      </ul>
+      <ul class="list">
+        <li class="list">
+          <a class="skillset" href="#">Javascript</a>
+          <a class="skillset" href="#">Ruby</a>
+          <a class="skillset" href="#">Html</a>
+          <a class="skillset" href="#">Css</a>
+        </li>
+      </ul>
+    </div>
 
-    <ul class="language">
-      <a class="icon" href="#"><img src="images/frameworks.png" alt="" /></a>
-      <li>
-        <a class="tech">Frameworks</a>
-      </li>
-    </ul>
-    <ul class="list">
-      <li class="list">
-        <a class="skillset" href="#">Bootstrap</a>
-        <a class="skillset" href="#">Ruby on Rails</a>
-        <a class="skillset" href="#">RSPec</a>
-        <a class="skillset" href="#">CapyBara</a>
-        <a class="skillset" href="#">Selenium</a>
-      </li>
-    </ul>
+    <div class="col4">
+      <ul class="language">
+        <a class="icon" href="#"><img src="images/frameworks.png" alt="" /></a>
+        <li>
+          <a class="tech">Frameworks</a>
+        </li>
+      </ul>
+      <ul class="list">
+        <li class="list">
+          <a class="skillset" href="#">Bootstrap</a>
+          <a class="skillset" href="#">Ruby on Rails</a>
+          <a class="skillset" href="#">RSPec</a>
+          <a class="skillset" href="#">CapyBara</a>
+          <a class="skillset" href="#">Selenium</a>
+        </li>
+      </ul>
+    </div>
 
-    <ul class="language">
-      <a class="icon" href="#"><img src="images/skills.png" alt="" /></a>
-      <li>
-        <a class="tech">Skills</a>
-      </li>
-    </ul>
-    <ul class="list">
-      <li class="list">
-        <a class="skillset" href="#">Codekit</a>
-        <a class="skillset" href="#">Github</a>
-        <a class="skillset" href="#">Codepen</a>
-        <a class="skillset" href="#">Gitlab</a>
-        <a class="skillset" href="#">Terminal</a>
-      </li>
-    </ul>
+    <div class="col4">
+      <ul class="language">
+        <a class="icon" href="#"><img src="images/skills.png" alt="" /></a>
+        <li>
+          <a class="tech">Skills</a>
+        </li>
+      </ul>
+      <ul class="list">
+        <li class="list">
+          <a class="skillset" href="#">Codekit</a>
+          <a class="skillset" href="#">Github</a>
+          <a class="skillset" href="#">Codepen</a>
+          <a class="skillset" href="#">Gitlab</a>
+          <a class="skillset" href="#">Terminal</a>
+        </li>
+      </ul>
+    </div>
   </section>
 
   <section class="form">
```

Comment:
- Great job using a single name input instead of `First name` and `Last name`.
- [ ] Please check the proportion of the left and right parts of the contact section. Currently, the left side is covering a little space making everything look a bit tight. Kindly update it according to the template. 🥸️
- [ ] Kindly add a line above the social icons as one of the objectives of this project is to understand how to parse Figma designs precisely. 😎️
- [ ] Please update the color inside the `textarea` element, and just like that perfection is around the corner. 🧐️

**Your Design**
![Screenshot from 2022-12-16 10-13-49](https://user-images.githubusercontent.com/17243325/208044736-2d46102d-faf9-4fda-9417-dd8d969314d4.png)

**Figma Design**
![Screenshot from 2022-12-16 10-14-23](https://user-images.githubusercontent.com/17243325/208044812-8fd9bc46-c72c-462a-a5a8-99036c93c127.png)